submissions/sprint_a/app.py

```diff
@@ -0,0 +1,14 @@
+from sprint_a import app
+
+data = {
+    "epithets": [],
+    "vocabulary": {}
+}
+
+@app.route('/')
+def generate_epithets():
+    return str(data["epithets"])
+
+@app.route('/vocabulary')
+def vocabulary():
+    return str(data["vocabulary"])
```

Comment:
views should return JSON using jsonify
jsonify both correctly converts native data types to JSON, it sets the correct HTTP headers